templates/repo/actions/runs_list.tmpl

```diff
@@ -1,14 +1,18 @@
 <div class="issue list">
 	{{range .Runs}}
-		<li class="item gt-df gt-py-3">
+		<li class="item gt-df gt-py-3 gt-ab">
 			<div class="issue-item-left gt-df">
 				{{template "repo/actions/status" .Status}}
 			</div>
-			<div class="issue-item-main gt-f1 gt-fc gt-df">
+			<div class="issue-item-main action-issue-item gt-f1 gt-fc gt-df gt-mr-3">
 				<div class="issue-item-top-row">
-					<a class="index gt-ml-0 gt-mr-2" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
+					<a class="index gt-ml-0 gt-mr-2" title={{.Title}} href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
 						{{- .Title -}}
 					</a>
+				</div>
+				<div class="desc issue-item-bottom-row gt-df gt-ac gt-fw gt-my-1">
+					<b>{{if not $.CurWorkflow}}{{.WorkflowID}} {{end}}#{{.Index}}</b>: {{$.locale.Tr "actions.runs.commit"}}
+					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a>&nbsp; {{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName | Escape}}
```

Comment:
```suggestion
					<a href="{{$.RepoLink}}/commit/{{.CommitSHA}}">{{ShortSha .CommitSHA}}</a> {{$.locale.Tr "actions.runs.pushed_by"}} {{.TriggerUser.GetDisplayName}}
```

Why `&nbsp;` and `Escape`? Neither of them seem right to appear ....

Reply:
> &nbsp;

This is actually copied, what i did here is to move the position of the whole branch label span, `&nbsp;` looks weird to me though, and i coule remove it, but not very sure what Escape does.

Reply:
> Why `&nbsp;` and `Escape`? Neither of them seem right to appear ....

This is actually copied, what i did here is to move the position of the whole branch label span, &nbsp; looks weird to me though, and i coule remove it, but not very sure what Escape does.